source/sync/data-model/update-schema.txt

```diff
@@ -114,10 +117,7 @@ the client-side object model. For more details, see relevant sections below.
    * - :ref:`Change property type but keep the name <schema-update-change-property-type-but-keep-name>`
      - Breaking
      - Breaking
-   * - :ref:`Change property from optional to required <schema-update-change-property-from-optional-to-required>`
-     - Breaking
-     - Breaking
-   * - :ref:`Change property from required to optional <schema-update-change-property-from-required-to-optional>`
+   * - :ref:`Change property's status between from optional nd required <schema-update-change-property-status>`
```

Comment:
Overall, I was (am) aiming for simplification of the page. It just felt so overwhelming, especially since so much of the content was duplicated. This seems like a good place to reduce the duplication.

Reply:
Alright, that's fair. There's still a word choice issue, though - we should pick either "between" or "from". (I can't seem to fix that on this thread since it's outdated.)